tests/test_common.py

```diff
@@ -50,6 +50,15 @@ def test_password(testdir):
     assert_frame_equal(df, tables[0].df)
 
 
+@skip_on_windows
```

Comment:
copied from ghostscript tests. nit: this should probably be an xfail, not a skip